R/run_stress_test.R

```diff
@@ -34,6 +38,8 @@
 #' @return NULL
 #' @export
 run_stress_test <- function(asset_type,
+                            data_path_project_specific,
+                            data_path_project_agnostic,
```

Comment:
I think it's rare that a function would have two arguments for input. Commonly you have only one. Maybe take a single path to the single parent directory containing two subdirectoreis -- one for project specific and the other one for project agnostic data?

Reply:
Also, the biggest concern is the output directory. Right now I think it's unclear to the user how to set outputs independently from inputs. That will make the function a lot easier to test, since we'll be able to send outputs to a self-destroyable directory like `tempdir()`.

Reply:
I would rather not combine  project agnostic and project independent files under a common subdirectory. This is for practical reasons:

Reply:
this way however people would have to pass also the name of the project to the function though, at least if we want to avoid that ST_INPUTS master has to be copy pasted into every single projects folder